lib/sequent/core/event_record.rb

```diff
@@ -5,10 +5,9 @@ module Sequent
   module Core
 
     module SerializesEvent
-
       def event
         payload = Sequent::Core::Oj.strict_load(self.event_json)
-        Class.const_get(self.event_type.to_sym).deserialize_from_json(payload)
+        Helpers::constant_get!(self.event_type).deserialize_from_json(payload)
```

Comment:
```ruby
module Foo; class Bar; end; end
Class.const_get('Foo::Bar'.to_sym) # throws NameError: wrong constant name Foo::Bar
```

So that's why we added the `constant_get!` function. But now it turns out that

```ruby
Class.const_get('Foo::Bar') # Returns the Foo::Bar class
```

does work (string argument instead of symbol). So we'll try to use that instead.